adc/python/adcPython/.doc/README.md

```diff
@@ -1,6 +1,18 @@
+# Python 3 Console Template Specific Documentation
 
+#ADC Python Sample
+
+This sample interacts with ADCs, in Python, through the Industrial I/O (IIO) sysfs interface.
+In this sample, the program reads the Voltage directly from the Toradex Device and print the correct voltage trough the path.
+
+In the 'docker-compose' file, it is shown the necessary path and what needs to be set.
```

Comment:
To check the available interfaces for your device, please check
 

the [UART (Linux)](https://developer.toradex.com/linux-bsp/application-development/peripheral-access/uart-linux) article.
 

And then update it on the `main.py` and `docker-compose.yml` files.